zeppelin-interpreter/pom.xml

```diff
@@ -52,6 +52,11 @@
   <dependencies>
 
     <dependency>
+      <groupId>commons-configuration</groupId>
```

Comment:
pom file change, because I move interpreter package of `zeppelin-zengine` to `zeppelin-interpreter`